Azure/azure-cli
# [Container] `az container export`: Fix export when identity is set

**Related command**


```sh
az container export
```

**Description**


When running the `az container export` command for a container that has/had managed identity setup on it, this would fail due to the identity object. This was due to a bug in our source code that failed converting the object to a dictionary before passing it to the `yaml` library for serialization. This fixes this.

resolves #24732
resolves #24731

**Testing Guide**


Create a container app, assign managed identity (system and/or user) and run the following command for the same

```sh
az container export
```

---

This checklist is used to make sure that common guidelines for a pull request are followed.
